modules/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/FileSpanExporter.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.common.InstrumentationScopeInfo;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.util.Collection;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Writes the span to a log file. This is mostly for testing where you can trace one request and all your spans
+ * will be written to a file. Tracing log file would be available in the logs directory.
+ */
+public class FileSpanExporter implements SpanExporter {
```

Comment:
The `opentelemetry-java` comes with `LoggingSpanExporter` already (see please https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters), why we need this one?

Reply:
This exporter appends the span data in the same log file, hence we created our own. Removed it for now